app/helpers/routing_helper.rb

```diff
@@ -2,37 +2,53 @@
 require 'json'
 
 module RoutingHelper
-  def self.seconds_to_minsec(sec)
+  def self.format_seconds_as_minsec(sec)
     format("%<minutes>.2d:%<seconds>.2d", minutes: sec / 60, seconds: sec % 60)
   end
 
-  def self.valid_coordinates(coordinates)
+  def self.coordinates(input, your_location)
+    return input if valid_coordinates?(input)
+    return your_location if input == "Your location"
+
+    Places::DESTINATIONS.dig(:"#{input}", :location)
```

Comment:
Nice method!

It seems as if `DESTINATIONS` should have string keys instead of symbols. You can archive this be replacing the `a: b` syntax (which, I think, is short for `:a => b`) with just `a => b` and don't need the symbol creation here :)

Reply:
You're absolutely right. Using symbols instead of strings in app/assets/constants/places.rb is inappropriate. The latest commit uses strings.